Makefile

```diff
@@ -1,5 +1,5 @@
 # VERSION defines the project version for the deploy scripts, not for bundles
-VERSION ?= main
+VERSION ?= 1.0.1
```

Comment:
please revert this change

Reply:
yes the reason was `make bundle` was broken it never picked up the right operator image so after fixing that if I don't align IMG tag to `1.0.1` every attempt to make bundle will generate diffs with operator image tag difference so to avoid this I used the same VERSION tag value similar to the bundle version docker image tag

Reply:
Still, the version should remain `main` by default to not break other make targets.
We run `VERSION=x.y.z make bundle`, or `IMG=(my operator image) make bundle` when needing to test a bundle. I did that just this morning to test your PR, for instance. It should work fine. It's just that we should not commit the generated files that point to the test version. If in doubt about what you would commit, just run another `make bundle` (without passing VERSION this time) to clean the generated bundle

Reply:
ok revert the version change

Reply:
w/o this change github bundle verify fail
https://github.com/netobserv/network-observability-operator/actions/runs/4224403226/jobs/7335311925